templates/base/head.tmpl

```diff
@@ -61,6 +61,7 @@
 	<meta property="og:description" content="{{MetaDescription}}">
 {{end}}
 	<meta property="og:site_name" content="{{AppName}}">
+	<link href="{{AppSubUrl}}/assets/css/swagger.css" rel="stylesheet">
```

Comment:
Can this be loaded conditionally?

Reply:
This approach is not right, which would cause unfixable conflicts.

Reply:
I think both the JS and CSS should be loaded via JS, e.g. similar to other renderers like Mermaid.

Reply:
I used a new method to dynamically load style files.